tests/suites/test_suite_camellia.data

```diff
@@ -13,16 +13,16 @@ camellia_encrypt_ecb:"0123456789abcdeffedcba98765432100011223344556677":"0123456
 Camellia-256-ECB Encrypt RFC3713 #1
 camellia_encrypt_ecb:"0123456789abcdeffedcba987654321000112233445566778899aabbccddeeff":"0123456789abcdeffedcba9876543210":"9acc237dff16d76c20ef7c919e3a7509":0
 
-Camellia-128-ECB Encrypt Perl EVP #1
+Camellia-128-ECB Encrypt Perl EVP #1 [#1]
```

Comment:
These seem to be a bit unnecessary. Please unify them so that all use the same [#n] format.

Reply:
Rejected as out of scope. The existing numbers are sometimes significant because they match numbers in a reference document. There are ~1400 duplicates and I don't have time to manually review them all.

Reply:
I can understand that we need unique descriptions, but having, for example:
- ARC4 Encrypt and decrypt 22 bytes in multiple parts 1 [#2]
- ARC4 Encrypt and decrypt 22 bytes in multiple parts 1 [#3]


when it should rather:
- ARC4 Encrypt and decrypt 22 bytes in multiple parts
- ARC4 Encrypt and decrypt 23 bytes in multiple parts 

makes a big difference.

Reply:
I fully agree that this is a bug, and thank you for doing the analysis. But it is out of scope of the current pull request, which solely aims to make test descriptions unique. I've filed https://github.com/ARMmbed/mbed-crypto/issues/271 to make these test descriptions correct.

Reply:
> solely aims to make test descriptions unique.

You did fix a few things on accident. Collateral damage. :)

Reply:
I decided that when two test cases had the same test data and the same test description, that absolutely had to be a mistake, which could either be that the same paragraph was copy-pasted more than needed or that the paragraph was copy-pasted but not edited when it should have been, meaning that the test data is not what was intended and we have missing coverage. The number of such cases was manageable so I went through them. The other cases are identical descriptions for differing test data, and while the test data may be wrong sometimes, in most cases I think it's only the description that's wrong.